values.yaml

```diff
@@ -94,6 +94,24 @@ expose:
 # If Harbor is deployed behind the proxy, set it as the URL of proxy
 externalURL: https://core.harbor.domain
 
+## set if you're using external services with a self signed cert
+# authCACert: |
+#   -----BEGIN CERTIFICATE-----
```

Comment:
We should keep the same style for all the certificates used in Harbor such as https://github.com/goharbor/harbor-helm/blob/master/values.yaml#L18. Maybe we can rename the `authCACert` to `uaaCACert` and refine the comment?

Reply:
Yes that makes sense to be able to look for it in an existing secret, I'll look to update that.

the reason I named it `authCACert` is that I assumed that at some point we might also support SSL for ldap/oauth/oidc and we could use this variable to provide the ca-cert for any of them, rather than needing a separate variable name each.

Reply:
Can we just keep the `uaaSecretName` and remove the `uaaCACert` as other secrets cannot be set as the content of the certificate?

Reply:
And please squash your commits